README.md

```diff
@@ -1 +1,44 @@
-# tf_datadog
\ No newline at end of file
+# terraform-datadog-monitor
+
+A Terraform module which contains a number of common configurations of monitors for Datadog.
+
+
+## Datadog monitors catalog
+
+- [load_average](https://github.com/cloudposse/terraform-datadog-monitor/tree/master/modules/load_average)
+    - Terraform module to provision Load Average Monitors in Datadog.
+
+## Usage
+
+- [load_average](https://github.com/cloudposse/terraform-datadog-monitor/tree/master/modules/load_average)
+
+Create monitor for all hosts in Datadog:
+
+```terraform
+module "global_monitors" {
+  source          = "https://github.com/cloudposse/terraform-datadog-monitor//modules/load_average"
```

Comment:
pin to `ref=master`

Reply:
@osterman, I didn't find the way how to use the `ref` or `tags` with subfolders